websites/V/VRCArena/dist/metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "KnuffelBeestje",
+    "id": "219155156569161731"
+  },
+  "url":
+    "www.vrcarena.com",
+  "description": {
+    "en": "VRCArena is a website created to provide VRChat assets like avatars, accessoires etc",
```

Comment:
Gebruik de officiële beschrijving:
"Download and upload various kinds of assets and tutorials for the different species found in the online multiplayer VR social game VRChat."

Reply:
Bedankt om dit even aan het licht te brengen. Had zelf niet door dat er een officiële beschrijving was. De aanpassingen zijn gemaakt.